pkgs/development/python-modules/flower/default.nix

```diff
@@ -1,22 +1,42 @@
-{ lib, buildPythonPackage, fetchPypi, Babel, celery, importlib-metadata, pytz, tornado, mock }:
+{ lib
+, buildPythonPackage
+, fetchPypi
+, Babel
+, celery
+, future
+, humanize
+, importlib-metadata
+, mock
+, pytz
+, tornado
+}:
 
 buildPythonPackage rec {
   pname = "flower";
   version = "0.9.4";
-  
+
   src = fetchPypi {
     inherit pname version;
     sha256 = "25782840f7ffc25dcf478d94535a2d815448de4aa6c71426be6abfa9ca417448";
   };
 
-  propagatedBuildInputs = [ Babel celery importlib-metadata pytz tornado ];
-  
+  # flower and humanize aren't listed in setup.py but imported
+  propagatedBuildInputs = [
+    Babel
+    celery
+    future
+    importlib-metadata
+    pytz
+    tornado
+    humanize
```

Comment:
[fancy]
should it go above `pytz` ?

Reply:
I forgot to sort, but it wasn't sorted before. I'm more concerned about the build failures than formatting :). doing small fixes on 10 different packages (and maintaining git practices) is a little painful